USER_GUIDE.md

```diff
@@ -153,6 +153,7 @@ print(response)
 - [Search](guides/search.md)
 - [Point in Time](guides/point_in_time.md)
 - [Using a Proxy](guides/proxy.md)
+- [Using Index Templates](guides/index_template.md)
```

Comment:
I think only Index Templates is enough :)

Reply:
@saimedhi This has been fixed and pushed.